docs/platform/changelog.md

```diff
@@ -4,8 +4,24 @@ description: What's new and improved in Grid.
 import Note from "@site/src/components/Note";
```

Comment:
we don't need this file.

can you replace its content to

```
### Don't edit, this file is autogenerated by `make build`
```

Reply:
done. thanks!